spaceship/lib/spaceship/test_flight/tester.rb

```diff
@@ -26,6 +66,13 @@ def self.find(app_id: nil, email: nil)
       return testers.first
     end
 
+    class << self
+      def factory(attrs)
+        attrs['statusModTime'] = Time.parse(attrs['statusModTime']) if attrs['statusModTime'].to_s.length > 0
+        return self.new(attrs)
```

Comment:
Let move away from this factory pattern.

This should be implemented by overriding the getter
```ruby
def status_mod_time
  value = raw_data['statusModTime']
  Time.parse(value) if value.to_s.length > 0
end
```